src/utils/githubEventHelpers.ts

```diff
@@ -48,7 +48,7 @@ export async function addIssueToGlobalIssuesProject(
 
   const response: any = await octokit.graphql(addIssueToGlobalIssuesProjectMutation);
 
-  return response.addProjectV2ItemById.item.id;
+  return response?.addProjectV2ItemById.item.id;
```

Comment:
I think this resolves https://sentry.sentry.io/issues/4239815868/?project=5246761&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=2&utc=true

Reply:
🤞